src/displayapp/screens/Calculator.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "displayapp/screens/Screen.h"
+#include "displayapp/apps/Apps.h"
+#include "displayapp/Controllers.h"
+#include "Symbols.h"
+
+namespace {
+  int64_t constexpr powi(int64_t base, uint8_t exponent) {
```

Comment:
```suggestion
  constexpr int64_t powi(int64_t base, uint8_t exponent) {
```
I didn't even know both ways were allowed! I think it reads nicer this way round?